src/vs/base/parts/quickinput/browser/media/quickInput.css

```diff
@@ -10,6 +10,7 @@
 	padding: 0 1px 1px 1px;
 	left: 50%;
 	margin-left: -300px;
+	-webkit-app-region: no-drag;
```

Comment:
Is there a non-webkit-prefixed form?

Reply:
this only applies to electron-based clients